package.json

```diff
@@ -148,7 +148,9 @@
     "rc-upload": "~3.3.4",
     "rc-util": "^5.1.0",
     "scroll-into-view-if-needed": "^2.2.25",
-    "warning": "^4.0.3"
+    "warning": "^4.0.3",
+    "react-dnd": "^11.1.1",
```

Comment:
Sorry, we won't depend on react-dnd in antd pacakge, which will bring a lot of bundle size.